### ISSUE TITLE
Need guidance for developing extensions for the new Editor

### ISSUE THREAD

**Issue by corliss** [mhutch/MonoDevelop.AddinMaker#73]
Closing as this is no longer needed.

**Comment by corliss** [comment]
We're trying to port this add-in to Visual Studio for Mac's New Editor.
The documentation for how to do this is scarce, we've gathered what we could from the initial announcement, the brief summary of developing with the new editor, and various blog posts. 

Could you please explain exactly how to register such an add-in with Visual Studio for Mac? We're creating a separate add-in for Mac, so we don't need it to be cross-platform with Windows; just the Mac instructions will suffice. 

The functionality we need to expose is basically of listening for new text editor views and hooking into the keyboard pipeline.

We're blocked on this; hope you can help.

Thanks!
